### ISSUE TITLE
Add templatetag for transition logs

### ISSUE THREAD

**Issue by florianm** [gizmag/django-fsm-log#40]
Would it make sense to provide a templatetag for the transition logs, or include a section in the README?

Here's a working example:

`myproject/myapp/subapp/templatetags/__init__.py`:
Don't forget to register the templatetags folder as module. I've placed the templatetags inside the nested subapp that has django_fsm transitions.

`myproject/myapp/templatetags/transitions.py`

```
from django import template
from django_fsm_log.models import StateLog
register = template.Library()

@register.inclusion_tag('tx_logs.html', takes_context=False)
def tx_logs(obj):
    """Renders the FSM transition logs for an object."""
    return {'logs': [log for log in StateLog.objects.for_(obj)]}
```

A very basic template `myproject/templates/tx_logs.html` - note my user model has a field `name`:

```
{% for log in logs %}
<div><strong>{{ log.timestamp|date:"r" }}</strong> {{ log.state }} by {{ log.by.name }}</div>
{% endfor %}
```

Feel free to place the template in a better place, just make sure to adjust the path in the inclusion tag accordingly.

In this example, I'm overriding the admin change form with a custom `myproject/myapp/templates/admin/change_form.html`, 
inserting the logs underneath the messages block. YMMV.

```
{% extends "admin/change_form.html" %}
{% load i18n admin_urls transitions %} {# load templatetag library transitions #}
{% block messages %}
    {{ block.super }}
    {% if original %}
    {% tx_logs original %} {# the object "original" has a django_fsm status field #}
    {% endif %}
{% endblock %}
```

As I'm overriding an admin change form, the object is available in the template as `original`.

Happy to send PR if appropriate.

**Comment by ticosax** [comment]
How does this play with #43, is there is overlapping ?

**Comment by florianm** [comment]
@ticosax #43 is a few months younger than my idea above, and looks more refined. 

My example is a bare-bones MWE displaying a list of state changes. I'm using it in production in Django 1.11.

#43 works for 1.7.7 (before the breaking changes and deprecations in 1.9 and later) and touches more internals, so care has to be taken when upgrading across Django versions. I reckon it will be easier to include (being an admin mixin), while my example requires adding a custom template. 

I'd be happy to see #43 merged and something along the lines of my example either included in the docs or as an actual template tag. Whatya think @ticosax?

**Comment by ticosax** [comment]
Thank you for you feedback.
That's my impression also, that #43 is easier to integrate in a existing project. I'll will first ask the submitter if he is willing to contribute again.

**Comment by ticosax** [comment]
Probably solved by #58

**Comment by florianm** [comment]
@ticosax Thanks, using it now in anger (and with pleasure)! Exactly what I needed, thanks!
